### PR TITLE
Fix paste autoscroll and add video compatibility

### PR DESCRIPTION
**Pull request by alessbelli**
also fixes that instance ordering causing a referenceerror which @mirip fixed

### CODE REVIEW

**Comment by alessbelli** [comment]
https://internal.bubbleapps.io/task/videos-uploaded-to-the-app-cannot-be-added-in-the-rich-text-editor-1593546858113x118562096632299520

https://internal.bubbleapps.io/task/pasting-into-rte-element-scrolls-back-to-top-of-page-1593042126009x756100798723129300

File: elements/AAC-850m6/update.js [hunk 1]
```diff
@@ -426,6 +436,22 @@ var update = function(instance, properties, context) {
 
       //add Quill container div to page
       instance.canvas.append(`<div id="${id}"></div>`);
+
+      // Hack the clipboard to bypass the annoying re-focus
+      // which centers the top of the Quill container after paste
+      const Clipboard = Quill.import('modules/clipboard')
```
Comment (g-bunny) [inline elements/AAC-850m6/update.js:442]: missing semicolons here?

**Review by g-bunny — COMMENTED** [review]
I guess semicolons aren't technically necessary, but we should either use them all throughout or never use them for consistency. Other than that, looks good to me

File: elements/AAC-850m6/update.js (hunk 1 above)
Comment (alessbelli) [reply elements/AAC-850m6/update.js:442]: yeah it's a bit inconsistent but our standard JS doesn't use semicolons, so I took the habit of removing them wherever I went. now this file may have some and be missing some but you're right that it mostly has them so I'll add them back for consistency

**Pull request closed without merging**